platforms/nuttx/cmake/upload.cmake

```diff
@@ -77,20 +78,33 @@ endif()
 
 string(REPLACE ";" "," serial_ports "${serial_ports}")
 
-add_custom_target(upload
-	COMMAND ${PYTHON_EXECUTABLE} ${PX4_SOURCE_DIR}/Tools/px_uploader.py --port ${serial_ports} ${fw_package}
-	DEPENDS ${fw_package}
-	COMMENT "uploading px4"
-	VERBATIM
-	USES_TERMINAL
-	WORKING_DIRECTORY ${PX4_BINARY_DIR}
-	)
+if(PX4_BOARD STREQUAL "nxp_tropic")
```

Comment:
hmm, let's find a better place for this board specific bit.

How about carrying these targets in `boards/nxp/tropic/cmake/upload.cmake`?

If teensy variants become a thing we could find somewhere shared.